TourMate/TourMate/Frontend/Plan/Views/EditPlanView/EditPlanViewModel.swift

```diff
@@ -95,6 +95,10 @@ class EditPlanViewModel<T: Plan>: PlanFormViewModel<T> {
 
     func updatePlan() async {}
 
+    func getTripLocation() -> Location {
+        Location()
+    }
```

Comment:
I think we should push up to planformviewmodel, since they exist in both add and edit